apps/src/javalab/Javalab.js

```diff
@@ -76,6 +76,10 @@ Javalab.prototype.init = function(config) {
   const onCommitCode = this.onCommitCode.bind(this);
   const onInputMessage = this.onInputMessage.bind(this);
 
+  Javalab.handleVersionHistory = this.studioApp_.getVersionHistoryHandler(
```

Comment:
for my information: is there a reason this is `Javalab.handleVersionHistory` instead of `const handleVersionHistory`?

Reply:
This is an excellent question... I noticed the distinction in AppLab and was trying to follow suit. That being said, I'm not sure I understand why! Happy to switch it and see if anything breaks.

Reply:
+1 I'd also suggest moving it to const. Also moving it up near where we're setting the other handlers (i.e. by `onInputMessage` and `onCommitCode`)

Reply:
Just saw this one- thanks!